physionet-django/user/templates/user/edit_trainings.html

```diff
@@ -0,0 +1,57 @@
+{% extends "user/settings.html" %}
+
+{% block title %}PhysioNet Trainings{% endblock %}
+
+{% block main_content %}
+<h1>PhysioNet Trainings</h1>
+  <hr>
+  <h4>Active trainings</h4>
+  <ul class="list-group list-group-flush">
+    {% for training in trainings %}
+      <li class="list-group-item d-flex align-items-center justify-content-between">
+        <div>
+          {{ training.training_type.name }} Training
+          {% include 'user/training_badge.html' %}
+        </div>
+        <a href="{% url 'edit_trainings_detail' training.id %}" class="btn btn-sm btn-primary" role="button">View</a>
+      </li>
+    {% empty %}
+      <p>There are no trainings.</p>
+    {% endfor %}
+  </ul>
+  <hr>
+  <h4>New training</h4>
+  {% lorem 1 p %}
```

Comment:
Provide a completion report indicating that you have finished the respective training course.